auth/src/main/java/com/firebase/ui/auth/util/GoogleApiHelper.java

```diff
@@ -0,0 +1,13 @@
+package com.firebase.ui.auth.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GoogleApiHelper {
+    private static final List<Void> IDS = new ArrayList<>();
```

Comment:
I think this would do better as an `AtomicInteger` and then your static function would just be wrapping `incrementAndGet()`